cl/assets/templates/base.html

```diff
@@ -54,6 +54,10 @@
 
   <link href="{% static "css/font-awesome.css" %}" rel="stylesheet">
 
+  <link href="{% static "css/app.css" %}" rel="stylesheet">
```

Comment:
I'm finding this a bit weird. We link to app.css, which imports tailwindcss. Gotta be a better way. I think Bots.law just has:

```
  {% tailwind_preload_css %}
  {% tailwind_css %}
```

Honestly not sure what the preload vs. regular do, but definitely simpler/cleaner?